lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1082,6 +1082,7 @@ def cla(self):
         # Disabling mouse interaction might have been needed a long
         # time ago, but I can't find a reason for it now - BVR (2012-03)
         #self.disable_mouse_rotation()
+        Axes.cla(self)
```

Comment:
@WeatherGod , curious (but not so important) question. Any reason why we're not using `super(Axes3D, self).cla()`? Since I'm on this line I could change to the newer method.